minimesos/src/main/java/com/containersol/minimesos/mesos/MesosAgentContainer.java

```diff
@@ -111,4 +116,13 @@ protected CreateContainerCmd dockerCommand() {
         envs.put("SERVICE_IGNORE", "1");
         return envs;
     }
+
+    @Override
+    public State getState() {
+        try {
+            return State.fromJSON(getStateInfoJSON().toString());
+        } catch (JsonParseException | JsonMappingException | UnirestException e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS1166)